File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        for i in range(len(numbers)):
            idx = bisect.bisect_left(numbers,target - numbers[i],lo = i+1)
            if idx != len(numbers) and numbers[idx] + numbers[i] == target:
                return [i+1,idx+1]

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        unordered_map<int,int>mp;
        for(int i = 0 ; i < numbers.size();i++){
            if(mp.find(target- numbers[i]) != mp.end())return {mp[target-numbers[i]]+1,i+1};
            mp[numbers[i]] = i;
        }
        return {-1,-1};
    }
};